stepper.md

```diff
@@ -1,64 +1,214 @@
 
-Also includes optional support for acceleration and deceleration of the motor.
+Includes optional support for acceleration and deceleration of the motor.
 
-Added in version 2.4.0.
+Also includes multiStepper support which allows groups of steppers to be simultaneously passed different ```to``` values and the duration of their movements will match. Up to five multiStepper groups can be created. The total number of steppers is still limited to 10.
 
 Example files:
  * The Stepper feature is include by default in [ConfigurableFirmata.ino](https://github.com/firmata/ConfigurableFirmata/blob/master/examples/ConfigurableFirmata/ConfigurableFirmata.ino).
  * [Example implementation](https://github.com/firmata/ConfigurableFirmata/blob/master/src/StepperFirmata.cpp) as a configurable Firmata feature class.
- * [Example of Stepper implementation in StandardFirmata](https://github.com/soundanalogous/AdvancedFirmata). *Note the dependency on the FirmataStepper class.*
 
 Protocol
 ---
 
-Stepper configuration
+**Stepper configuration**
 
 *Note: `stepDelay` is the the number of microseconds between steps. The default
 value is 1us. You can change the delay to 2us (useful for high current stepper
 motor drivers). Additional delay values can be added in the future.*
 ```
-0  START_SYSEX                       (0xF0)
-1  Stepper Command                   (0x72)
-2  config command                    (0x00 = config, 0x01 = step)
-3  device number                     (0-5) (supports up to 6 motors)
-4  stepDelay | interface             (upper 4 bits = step delay:
-                                        0000XXX = default 1us delay [default]
-                                        0001XXX = 2us delay
-                                        additional bits not yet used)
-
-                                     (lower 3 bits = interface:
-                                        XXXX001 = step + direction driver
-                                        XXXX010 = two wire
-                                        XXXX100 = four wire)
-5  steps-per-revolution LSB
-6  steps-per-revolution MSB
-7  motorPin1 or directionPin number  (0-127)
-8  motorPin2 or stepPin number       (0-127)
-9  [only when interface = 0x04] motorPin3 (0-127)
-10 [only when interface = 0x04] motorPin4 (0-127)
-11 END_SYSEX                         (0xF7)
-```
-
-Stepper step
-```
-0  START_SYSEX          (0xF0)
-1  Stepper Command      (0x72)
-2  config command       (0x01)
-3  device number        (0-5)
-4  direction            (0-1) (0x00 = CW, 0x01 = CCW)
+0  START_SYSEX                                (0xF0)
+1  Stepper Command                            (0x62)
+2  config command                             (0x00 = config)
+3  device number                              (0-9) (Supports up to 10 motors)
+
+4  interface                                  (upper 3 bits = wire count:
+                                                010XXXX = two wire
```

Comment:
Oops, that's one of 'em but there is also a [two wire non-driver configuration](https://www.arduino.cc/en/Reference/StepperBipolarCircuit). I'll fix that.